MAUI/DateTimePicker/overview.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: About .NET MAUI Date Time Picker control | Syncfusion
+description: Learn here about the overview of Syncfusion .NET MAUI Date Time Picker (SfDateTimePicker) control, its basic features, and its functionalities.
+platform: maui
+control: SfDateTimePicker
+documentation: ug
+---
+
+# Overview of .NET MAUI Date Time Picker (SfDateTimePicker)
+
+Syncfusion .NET MAUI Date Time Picker (SfDateTimePicker) provides a way to pick a date, time or date and time. The Date Time Picker also enables you to set date ranges, date and time formats, and to customize its dialog appearance by configuring its header, footer, and more.
+
+## Key features
+
+* **Header view**: The .NET MAUI Date Time Picker allows the user to add and customize header text.
+
+* **Column header view**: The .NET MAUI Date Time Picker supports a customizable column header view to separate headings for each column.
+
+* **Footer view**: Provides validation buttons (OK and Cancel) in the footer view. The footer text and background color can be customized.
+
+* **Selection view**: Selection view is used to show the selected item and it can be customized.
+
+* **Date format**: Offers 8 predefined formats to represent the value of the date in different string formats.
+
+* **Time format**: Offers 9 predefined formats to represent the value of the time.
+
+* **Picker mode**: Show the picker in a Popup with a dialog mode. The relative dialog mode is used to align the picker in a specific position.
+
+* **Date restriction**: Restrict the selection of date items beyond the specified minimum and maximum dates.
+
+* **Date intervals**: The date values can be populated with intervals for days, months, and years, individually.
+
+* **Time intervals**: The time values can be populated with intervals for hours, minutes, and seconds, individually.
```

Comment:
* **Time intervals**: The time values can be populated individually with intervals for hours, minutes, and seconds.

Reply:
Content modified.